plugins/zns/zns.c

```diff
@@ -223,7 +223,20 @@ static int zone_mgmt_send(int argc, char **argv, struct command *cmd, struct plu
 	if (fd < 0)
 		return errno;
 
-	if (cfg.data_len) {
+	if (!cfg.zsa) {
+		fprintf(stderr, "zone send action must be specified\n");
+		err = -EINVAL;
+		goto close_fd;
+	}
+
+	if (cfg.zsa == 0x10){
```

Comment:
Define it and use it instead of 0x10.

Reply:
@MatiasBjorling forgot to modify as NVME_ZNS_ZSA_SET_DESC_EXT. ! thanks